recipes-core/packagegroups/packagegroup-ni-runmode.bb

```diff
@@ -24,6 +24,7 @@ RDEPENDS:${PN} = "\
 	libyaml \
 	linux-firmware-radeon \
 	lldpd \
+	lvrt-logging \
```

Comment:
This logging is not specific to lvrt. Rename to `nilrt-logging` or something else.

Reply:
I have fixed this now.